buildspec_windows_create_manifest.yml

```diff
@@ -0,0 +1,53 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - echo "Creating manifest for aws-for-fluent-bit Windows image"
+      - yum update -y && yum upgrade -y
+      - yum install jq -y
+  build:
+    commands:
+      - |
+        export DOCKER_CLI_EXPERIMENTAL=enabled
+        declare -A os_releases=( ["windows2019"]="ltsc2019" ["windows2022"]="ltsc2022")
+        
+        # Push the image to ECR with corresponding architecture as the tag.
+        aws ecr get-login-password --region ${AWS_REGION}| docker login --username AWS --password-stdin ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com
+        
+        # Find the applicable windows versions.
+        cd $CODEBUILD_SRC_DIR
+        versions=$(cat windows.versions | jq -c -r '.windows[].version')
+        latest_version=""
+        
+        # Create and push the image manifests.
+        while read -r version; do
```

Comment:
should this be `versions`?

Reply:
These would be the versions defined in `windows.versions` file. The same would have been built in prior stages.

Reply:
Cool but the variable that the file is read into above is called `versions` if I am reading this correctly

Reply:
So we will feed all the versions to the loop and in each iteration, we would be reading one version in the variable `version`.

Basically, `versions` would be something like below-
```
2.26.1
2.26.2
```

`read -r version` would read each line in `$version` on each iteration.